dev/ci/contrib.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -ex
+
+source dev/ci/exports.sh
+
+get-artifact-coq-with-fallback
+
+# considering this just calls one of the ci-* scripts maybe we should skip the Makefile?
```

Comment:
This could probably be done by sourcing exports.sh and calling get-artifact-coq-with-fallback in ci-common.sh, but it's not as uniform as having a contrib.sh.